README.md

```diff
@@ -11,6 +11,42 @@ The Deployment Driver consists of following components:
 - The web UI providing the user a way to interact with the deployment engine/server
 - Nginx web server and reverse proxy serving the installer web UI and proxy-ing API requests to installer engine
 
+## Local Development
+
+### Setup
+
+To setup for local development you will need to:
+- Have an Azure subscription and Resource Group created
+- Build the server
```

Comment:
Just as a side note, building is not always required, running `go run .` in the server source folder works too. 
We can include that as a side note or as a "advanced setup: for those who dare because it means the `.env` file needs to be in the source directory and the paths need to be adjusted.